easybuild/easyconfigs/h/Horovod/Horovod-0.18.2-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonBundle'
+
+name = 'Horovod'
+version = '0.18.2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
Include TensorFlow version in `versionsuffix`?

```python
local_tf_version = '2.0.0'
versionsuffix = '-TensorFlow-%s-Python-%%(pyver)s'
...
dependencies = [
    ...
    ('TensorFlow', local_tf_version, '-Python-%(pyver)s'),
]

Reply:
Like this?

PS: Could the page https://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html be updated to include something like:

For python3 use:
`pip install --user GitPython keyring keyrings.alt`

For python2 use:
`pip install --user 'GitPython<3.0' 'keyring<19.0' keyrings.alt`

I've reset my user env for testing and had to find that again because the info is a) scattered across the doc page and b) newer versions don't work with Python2

Reply:
Changes in this PR look good.

Are you up for issuing a PR to update the docs?
See https://github.com/easybuilders/easybuild/blob/develop/docs/Integration_with_GitHub.rst